### ISSUE TITLE
Training issue with DirectML

### ISSUE THREAD

**Issue by JohnAMacDonald** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1067]
Hello!

I just downloaded RVC0813AMD_Intel.7z from HuggingFace, great work on the release by the way!

I have an AMD RX 6750 XT 12 GB GPU and when training everything works and it uses my AMD GPU as in Task Manager it shows my GPU usage going to ~80% when extracting pitch with rmvpe_gpu, but when I get to the button "Train Model" and hit it the CMD window say's "NO GPU DETECTED: falling back to CPU - this may take a while". For the GPU Index I put "0" without quotes, I've also put "0-1-2" without quotes, but it still say's the same error.

Any tips/help? Thanks!

**Comment by TheWonderfulTartiflette** [comment]
It does exactly this to me as well, GPU used only when extracting pitch with rmvpe_gpu, though I am on an Intel Arc card. Maybe there's something wrong with the initial AMD/Intel release?
And for my part, I don't even see the little window in teh webui with "Enter GPU Index(es),separated by (Example: 0-1-2 to select card 1, 2 and 3" and "GPU Information", it's just blank :
![image](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/44057002/f56f67f2-8c00-4536-884d-b929b633aef2)

**Comment by JohnAMacDonald** [comment]
This is what my "Train" tab looks like directly after booting up with an AMD GPU. Posting in case it can be helpful on fixing the issue :)

![RVC Train Tab](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/62683266/8aa66eea-b0fa-4f33-9de5-db39490398e9)

**Comment by TheWonderfulTartiflette** [comment]
Yeah you've got the same problem as me, look at this screenshot I got from a guy that uses an NVIDIA GPU:
![Screenshot_20230824-083856.png](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/44057002/00fb4222-d41f-48c0-94a9-8942a5e826af)

**Comment by haseebmohammad1991** [comment]
Same error on AMD RX590 GME No GPU information
![image_2023-08-24_182003289](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/90326609/52cb55fb-ca83-424d-89d4-22fda51376bd)
Also NO GPU DETECTED
![image_2023-08-24_182047699](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/90326609/a532b735-5c1c-4898-8d2d-64aa3db226d5)

**Comment by volescoff** [comment]
I'm experiencing the exact same problem. The UI element for selecting a GPU under step 2b is missing and when using the "one-click-training" button or the "train model" button the command line states no gpu detected. Glad I'm not the only one with this problem. Hope it gets fixed.

CPU: 5 5600
GPU: 6800xt
OS: windows 11

**Comment by Acrivec** [comment]
7900 XTX on W10 also not detected.

**Comment by mixern6** [comment]
I have the same problem with RX 6800 XT and the most strange thing is that it utilizes the CPU only by 65-66%

**Comment by RVC-Boss** [comment]
AMD GPU trained is not supported now. We are trying to support it in the future versions.

**Comment by JohnAMacDonald** [comment]
> 

Sad to hear, must have misinterpreted that AMD training was supported.

Do you guys have a rough ETA of when AMD training might be implemented? :)

Thanks in advance!

**Comment by TheWonderfulTartiflette** [comment]
@RVC-Boss what about Intel GPUs?

**Comment by alanwang233233** [comment]
Hello, I meet the same issue yesterday, I wonder if AMD GPU Training is supported now?

**Comment by Acrivec** [comment]
> Hello, I meet the same issue yesterday, I wonder if AMD GPU Training is supported now?

Last commit 2 years ago, what do you think

**Comment by alanwang233233** [comment]
Alright. I'm sorry to hear that. I'm trying ZLUDA these days. I wish it could work.😂2026年2月4日 上午7:09，Acrivec ***@***.***>写道：Acrivec left a comment (RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1067)

Hello, I meet the same issue yesterday, I wonder if AMD GPU Training is supported now?

Last commit 2 years ago, what do you think

**Comment by KimeruReru** [comment]
guess we won't have news on this...

**Comment by alanwang233233** [comment]
maybe we can try rocm on linux

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
